app/scripts/services/brackets.coffee

```diff
@@ -9,6 +9,13 @@
 ###
 angular.module('edudashAppSrv').service 'bracketsSrv', ->
 
+  getBrackets: (metric) ->
```

Comment:
I think coloring the charts is still business logic, and the numbers have to match the ones used in `getBrackets` to be valid, so it would be nice if they weren't duplicated.